app/src/main/java/com/wittyapp/MyApplication.kt

```diff
@@ -0,0 +1,19 @@
+package com.wittyapp
+
+import android.app.Application
+import com.facebook.FacebookSdk
+import com.facebook.appevents.AppEventsLogger
+
+/**
+ * Created by bett on 8/22/17.
+ */
+
+class MyApplication: Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+
+        FacebookSdk.sdkInitialize(applicationContext)
```

Comment:
Add log messages before calling external SDK APIs so that we know if they take a long time, our app hasn't hung.